platform/javascript/export/export.cpp

```diff
@@ -584,18 +569,21 @@ Error EditorExportPlatformJavaScript::export_project(const Ref<EditorExportPrese
 
 	// Save a favicon that can be accessed without waiting for the project to finish loading.
 	// This way, the favicon can be displayed immediately when loading the page.
-	Ref<Image> favicon;
-	const String favicon_path = String(GLOBAL_GET("application/config/icon")).strip_edges();
-	if (!favicon_path.is_empty()) {
-		favicon.instance();
-		const Error err = favicon->load(favicon_path);
-		if (err) {
-			favicon.unref();
+	if (export_icon) {
+		Ref<Image> favicon;
+		const String favicon_path = String(GLOBAL_GET("application/config/icon")).strip_edges();
+		if (!favicon_path.is_empty()) {
+			favicon.instance();
+			const Error err = favicon->load(favicon_path);
```

Comment:
this (and the splash above), prints:
`Loaded resource as image file, this will not work on export:`
do we have an editor-way to do that?
Do I need to read the file, and call load_png_from_buffer? Or maybe I should just copy it, as is (hoping it's valid)?

Reply:
You can probably try to strip the `res://` path, or use `ImageLoader` directly:
```cpp
Error Image::load(const String &p_path) {
#ifdef DEBUG_ENABLED
        if (p_path.begins_with("res://") && ResourceLoader::exists(p_path)) {
                WARN_PRINT("Loaded resource as image file, this will not work on export: '" + p_path + "'. Instead, import the image file as an Image resource and load it normally as a resource.");
        }
#endif
        return ImageLoader::load_image(p_path, this);
}
```

This warning is semi-bogus though, @reduz added it because too many users were shooting themselves in the foot with that, but it's also raised on valid use cases.

Reply:
Thanks, using `ImageLoader` does the trick.